shared/profile/edit-profile/container.tsx

```diff
@@ -32,7 +32,6 @@ const mergeProps = (stateProps, dispatchProps) => ({
   location: stateProps.location,
   onCancel: dispatchProps.onCancel,
   onSubmit: dispatchProps.onSubmit,
-  title: 'Edit Profile',
```

Comment:
This adds the title on mobile
<img width="431" alt="image" src="https://user-images.githubusercontent.com/11968340/61079304-8b88fd00-a3f0-11e9-9b6c-daa469e60953.png">
However it doesn't need to be added back to the type, this HOC adds it https://github.com/keybase/client/blob/d77e8b873b5fd1c4fc8988416128987e7d9b5944/shared/profile/edit-profile/index.tsx#L117

Maybe it would be less confusing if we required the title. Anyway, for now can we add it back?

Reply:
oops yes